python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -559,7 +556,7 @@ def __new__(
     ):
         check.invariant(run_ids != [], "When filtering on run ids, a non-empty list must be used.")
 
-        return super(RunsFilter, cls).__new__(
+        return super().__new__(
```

Comment:
remove all the `check` calls below since the generated constructor we are calling with `super().__new__` will do them